packages/pipelines/pipeline-loader/CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # @ucdjs/pipelines-loader
 
+## [0.0.1-beta.6](https://github.com/ucdjs/ucd/compare/@ucdjs/pipelines-loader@0.0.1-beta.5...@ucdjs/pipelines-loader@0.0.1-beta.6) (2026-02-19)
+
+
+*No significant changes*
+
+##### &nbsp;&nbsp;&nbsp;&nbsp;[View changes on GitHub](https://github.com/ucdjs/ucd/compare/@ucdjs/pipelines-loader@0.0.1-beta.5...@ucdjs/pipelines-loader@0.0.1-beta.6)
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)